File: java/TestOpenAPIKey.java
package com.example.openapi;
import com.fasterxml.jackson.databind.ObjectMapper;
public class TestOpenAPIKey {
    public static void main(String[] args) {
        String API_KEY = "bgqbanjfUqPKihJ2FDhWs1h3JdaECZS0";
        String API_SECRET = "5vy5Oy7NFxoLNz8kuUDNCrBE2inMwf4mTm21SiRbj1HuNISHw73xQeubhxnrJ6PN";
        String BASE_URL = "http://10.0.0.132";       
        String RID = "tushuguan";
        String GET_URL = "/openapi/asset/connection/getData?rid=" + RID;        
        System.out.println("=== 发送GET请求 ===");
        try (OpenAPIKeyClient client = new OpenAPIKeyClient(BASE_URL, API_KEY, API_SECRET)) {
            Object result = client.get(GET_URL);            
            ObjectMapper objectMapper = new ObjectMapper();
            String jsonResult = objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(result);
            System.out.println(jsonResult);
            
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: python/TestOpenAPIKey.py
import sys
import json
sys.path.append(os.path.abspath('./venv'))
from OpenAPIKeyClient import OpenAPIKeyClient
def main():
API_KEY = "bggbanjUqGPkIh2FDWslh3JdEaCZSO"
API_SECRET = "5vySoyNFXoLnZ8kuJDNCRbE2inMwf4fmT21sIRbJ1hNISIwH73xQeubhxnJ6PN"
BASE_URL = "http://10.0.0.132"
RID = "tushuguan"
GET_URL = f"/openapi/asset/connection/getData?rid={RID}"
print("=== 发送GET请求 ===")
try:
client = OpenAPIKeyClient(BASE_URL, API_KEY, API_SECRET)
result = client.get(GET_URL)
print(json.dumps(result, indent=2, ensure_ascii=False))
except Exception as e:
print(f"请求失败: {str(e)}")
import traceback
traceback.print_exc()
finally:
try:
client.close()
except:
pass
if _name_ == "_main_":
main()